torchvision/transforms/transforms.py

```diff
@@ -370,36 +395,57 @@ def __repr__(self):
             format(self.padding, self.fill, self.padding_mode)
 
 
-class Lambda(object):
+class Lambda(torch.nn.Module):
     """Apply a user-defined lambda as a transform.
 
+    .. Note::
+        This class exposes ``lambd`` attributes as its own attributes:
```

Comment:
Question: do we want to mention it in the doc? Does this make it less of an implementation detail?

Reply:
Maybe we can rephrase to say that custom ops wrapped by Lambda can behave similarly to original custom op and we do not have to use `op.lambd` to access attributes.